deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -285,75 +285,106 @@ spec:
     quickly get up and running with this powerful concept. Federation is a key
     part of any Hybrid Cloud capability.
 
-    **Important Note**: This operator is purely for prototyping purposes and not
+    **Important Note**: 
+    * This operator is purely for prototyping purposes and not
     supported by any upstream community at this time.
 
-
-    **Important Note**: Currently, while in preview, this operator does not
+    * Currently, while in preview, this operator does not
     support automatic upgrades. You must remove the old version of the operator
     manually before installing a new version.
 
 
-    ## Using KubeFed
-
-    This deploys KubeFed in a namespace-scoped as well as in a cluster-scoped fashion.
-    For example, if you subscribe to this package in namespace `my-namespace`,
-    KubeFed will be deployed to manage objects _only_ in `my-namespace` in
-    the host cluster and target clusters.
-
-    After installing KubeFed instance, you will use the `kubefedctl` command
-    line tool to do two types of configuration:
-      
-    - Configure federation to connect to your target clusters with `kubefedctl join`
-    - Create new federation APIs for the API types you want to federate with `kubefedctl enable`
-
-
-    **Important Note**: To add federation capability to your namespace, you are
-    only required to install kubefed in the hosting cluster; there is no need
-    to install kubefed in the target clusters (and no effect gained from
-    doing so).
+    ## Start using KubeFed
 
+    This operator deploys KubeFed in a namespace-scoped as well as in a cluster-scoped fashion. For example, if you subscribe to this package
+    in namespace `my-namespace`, KubeFed will be deployed to manage objects _only_ in `my-namespace` in the host cluster and target clusters.
+    
+    ### KubeFed deployment
+    In order to deploy kubefed, you need to follow these two steps:
+
+    * Install the operator in a single `Namespace` or in All `Namespaces` based on the operator Installation Mode.
+    ```
+    $ kubectl get po -n <namespace>
+      NAME                               READY   STATUS    RESTARTS   AGE
+      kubefed-operator-db58d9b89-8694x   1/1     Running   0          15s
+
+    ```
+    * Create the `kubefed` Custom Resource to drive the installtion of `KubeFed` setting the scope to be either `Namespace` or `Cluster`. 
```

Comment:
`installation` vs. `installtion`

Reply:
setting the scope to be either `Namespaced`

Reply:
Fixed